src/anm/media/video.js

```diff
@@ -47,7 +47,7 @@ Video.prototype.load = function(elm, player) {
     var me = this;
     ResMan.loadOrGet(player.id, me.url,
         function(notify_success, notify_error, notify_progress) { // loader
-            var url = me.url;
+            var url = Array.isArray(me.url) && me.url.length > 0 ? me.url[0] : me.url;
```

Comment:
Frankly, for me extra symbols make it harder to read code in such simple cases. But since it's your codebase, I will follow your code style.